faker/providers/bank/fi_FI/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .. import Provider as BankProvider
+
```

Comment:
could you add a blank line here for PEP8?

Reply:
done

Reply:
I also added tests, and they run fine on my local machine
> python setup.py test 
results in all green. Am I missing something here?